guessNumber.js

```diff
@@ -0,0 +1,41 @@
+const readline = require('readline');
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+    return new Promise((resolve, reject) => {
+        rl.question(questionText, resolve);
+    });
+}
+
+start();
+
+async function start() {
+    console.log("Let's play a game where I (the computer) think of a number in a range that you give me and you (the human) try to guess it.")
+
+    await ask("Press ENTER when you're ready. ");
+    lowerRange = parseInt(await ask("Please give me a starting point. "));
```

Comment:
You should use the 'let' keyword when declaring and initializing a variable for the first time so to avoid creating accidental global variables